doc/user_manual/postprocessor.tex

```diff
@@ -109,6 +112,22 @@ \subsubsection{BasicStatistics}
     \item \textbf{skewness}: skewness
     \item \textbf{kurtosis}: excess kurtosis (also known as Fisher's kurtosis)
     \item \textbf{samples}: the number of samples in the data set used to determine the statistics.
+    \item \textbf{sharpeRatio}: the Sharpe Ratio, measures the performance of an investment. It is defined as the returns of the investment, divided by the standard deviation of the investment. 
+    \item \textbf{sortinoRatio}: the Sortino ratio, measures the risk-adjusted return of an investment asset. Discounts the excess return of a portfolio above a target threshold by the volatility of downside returns. If this quantity is inputted as \textit{sortinoRatio} the threshold for separate upside and downside value will assign as $0$. Otherwise the user can specify this quantity with a parameter \xmlAttr{threshold='X'}, where the \xmlAttr{X} represents the requested threshold \xmlAttr{median} or \xmlAttr{zero}.
```

Comment:
I would create a PostProcessor ad hoc for those indices (since later we want to move those in TEAL).

Can you do it?

Reply:
I create a group in the PP, is that OK?